fastcrypto/src/secp256k1/recoverable.rs

```diff
@@ -12,7 +12,7 @@
 //! let kp = Secp256k1RecoverableKeyPair::generate(&mut thread_rng());
 //! let message: &[u8] = b"Hello, world!";
 //! let signature = kp.sign(message);
-//! assert!(kp.public().verify(message, &signature).is_ok());
+//! assert_eq!(&signature.recover(message).unwrap(), kp.public());
```

Comment:
The doctest is also changed to show a public key recovery rather than a verification.